src/high-level/lapack-templates.lisp

```diff
@@ -23,18 +29,18 @@
                ((assertion (cl:= k brows))
                 (assertion (or (not target) (equal (shape target) (list m n)))))
              (let ((ta
-                     (if (eql :row-major (layout a))
-                         (case transa
+                    (if (eql :row-major (layout a))
+                      (case transa
```

Comment:
Indentation

Reply:
I'm not exactly sure what the problem was here, but I took a guess. :)

Reply:
```
(if thing
    then
    else)
```
vs
```
(if thing
  then
  else)
```
The former is preferred.